WordPress/gutenberg
# [RNMobile] [Embed block] Fix content disappearing on Android when switching light/dark mode

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3859





## Description


As it was done for the device rotation action, I'm adding to prop `key` of the SandBox component the value of the light/dark mode, this way when this device setting changes it will recreate the WebView to prevent the content from disappearing.

## How has this been tested?




1. Open the app on Android (connect the app to a local Metro server as the previews only work in development mode)
2. Open a post/page
3. Add an embed block
4. Add a valid embed URL
5. Observe that the embed preview is displayed
6. Switch to light or dark mode
7. Observe that the embed preview is still displayed

## Screenshots 

N/A

## Types of changes




Bug fix

## Checklist:
